db/seeds.rb

```diff
@@ -21,14 +21,20 @@
   ["Benin Airport", "Benin"],
   ["Benue Airport", "Benue"],
   ["Malawi Airport", "Malawi"],
+  ["Malasia Airport", "Malasia"],
   ["Ghana Airport", "Ghana"]
 ]
 airport.each do |name, location|
   Airport.find_or_create_by(name: name, location: location)
 end
 
-300.times do
-  Flight.find_or_create_by(destination_id: rand(1..6), departure_id: rand(1..6),
+150.times do
+  Flight.find_or_create_by(destination_id: rand(1..4), departure_id: rand(5..8),
+  departure_date: Time.zone.now + (rand(5..7) * 86400) + rand(3600..70000),
+  airline_id: rand(1..9), price: rand(10000..90000))
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Separate every 3 digits in the integer portion of a number with underscores(_).